From e4daf4def996e216fb8f6c00f2557a8f816b981a Mon Sep 17 00:00:00 2001
From: Viacheslav Bocharov <adeep@lexina.in>
Date: Thu, 6 Jun 2024 14:31:12 +0300
Subject: [PATCH 37/39] dt-bindings: arm: amlogic: add binding for JetHome
 JetHub D2

JetHome JetHub D2 is a home automation controller, based on Amlogic S905X3 SoC

Signed-off-by: Viacheslav Bocharov <adeep@lexina.in>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 1c1094cd6b77..7cd9d268c3b2 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -195,6 +195,7 @@ properties:
               - hardkernel,odroid-c4
               - hardkernel,odroid-hc4
               - haochuangyi,h96-max
+              - jethome,jethub-j200
               - khadas,vim3l
               - seirobotics,sei610
           - const: amlogic,sm1
-- 
2.45.2

